lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -244,7 +244,7 @@ func (handler *ContainersHandlersImpl) RemoveContainerHandler(params containers.
 func (handler *ContainersHandlersImpl) GetContainerInfoHandler(params containers.GetContainerInfoParams) middleware.Responder {
 	defer trace.End(trace.Begin("Containers.GetContainerInfoHandler"))
 
-	container := exec.ContainerInfo(params.ID)
+	container := exec.Containers.Container(params.ID)
```

Comment:
Yeah point taken. I initially changed it to `Get()` but didn't want to introduce even more changes to this PR. I can follow up with another PR that changes the names of the methods on the container cache.